src/chi-vue/src/views/DataTable/ClientSide/DataTableClientView.vue

```diff
@@ -96,10 +90,7 @@
         <template v-slot:end>
           <div class="chi-toolbar__actions-desktop">
             <DownloadButtonIcon />
-            <ColumnCustomization
-              @chiColumnsReset="(e) => chiToolbarColumnsReset(e)"
-              :columnsData="toolbar.columnsData"
-            />
+            <ColumnCustomization @chiColumnsReset="(e) => chiColumnsReset(e)" :columnsData="toolbar.columnsData" />
```

Comment:
I think this works too. Can you please check?
```suggestion
            <ColumnCustomization @chiColumnsReset="chiColumnsReset" :columnsData="toolbar.columnsData" />
```